### DISCUSSION TITLE
Learn Data Visualization: Replicating Cara Thompson's Work

### DISCUSSION THREAD

**Discussion by tongakuot** [has2k1/plotnine#821] [2024 Plotnine Contest]
### Authors

Alier Reng

### Links

https://github.com/tongakuot/ggplot2-py/blob/main/BarBeRights.ipynb

### Full description

This tutorial aims to reproduce **Cara Thompson's work** created with the ggplot2 package. I aim to showcase how to produce a similar plot using plotnine, a Python version of ggplot2 developed by Posit. Following this tutorial, you will learn how to leverage plotnine to achieve the same high-quality visualizations in Python, bridging the gap between R and Python for data visualization.
![p9_img](https://github.com/has2k1/plotnine/assets/77814470/b03a0958-da8d-4ec7-8500-69c41b1c4037)

### Code repository

https://github.com/tongakuot/ggplot2-py/blob/main/BarBeRights.ipynb

**Comment by has2k1** [comment]
![bbr](https://github.com/user-attachments/assets/ad699cca-6485-4065-b687-f53cbc074c65)

**Comment by has2k1** [comment]
@tongakuot, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
